src/modules/modules.py

```diff
@@ -102,6 +103,40 @@ def __init__(self, vocab, text_field_embedder, num_highway_layers, phrase_layer,
 
         initializer(self)
 
+    def get_cove_layer_embeddings(sent, sent_embs, task_sent_embs):
```

Comment:
nice, but minor nit: add `_` in front of the method name, i.e. `_get_cove_layer_embeddings`